taskipy/task_runner.py

```diff
@@ -1,9 +1,12 @@
+import signal
 import subprocess
 from pathlib import Path
 from typing import List, Union
 
-from taskipy.task import Task
 from taskipy.pyproject import PyProject
+from taskipy.task import Task
+
+CATCHABLE_SIGNALS = {signal.SIGTERM}  # Add more signals if needed
```

Comment:
Taskipy aims to be as transparent as possible. As such, I would rather that by default we allow all signals, so no need for `CATCHABLE_SIGNALS` ;)

Reply:
That's what I did first but the issue is it breaks almost all tests (`FAILED (failures=12, errors=3)`). Maybe we could add more signals to this list but `signal.Signals` seems to have too many signals.

Reply:
I see. then, in that case, let's only treat `SIGTERM` atm. What I mean by that is that we can remove the catchable signal set and just treat sigterm specifically in our code :)